lib/routes/ehentai/tag.js

```diff
@@ -1,10 +1,13 @@
+const querystring = require('querystring');
 const EhAPI = require('./ehapi');
 
 module.exports = async (ctx) => {
     const page = ctx.params.page ? parseInt(ctx.params.page) : 0;
     const tag = ctx.params.tag;
-    const bittorrent = ctx.params.bittorrent || false;
-    const items = await EhAPI.getTagItems(ctx.cache, tag, page, bittorrent);
+    const routeParams = querystring.parse(ctx.params.routeParams);
```

Comment:
The `querystring` API is considered Legacy. While it is still maintained, new code should use the `URLSearchParams` API instead.
docs: https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams